src/hooks/useForm.ts

```diff
@@ -31,7 +33,7 @@ const useForm = ({ isSignUp }: SignFormProps) => {
       password: password,
     }
 
-    if (isSignUp) {
+    if (mode === 'signup') {
```

Comment:
```suggestion
    if (mode === FormType.SignIn) {
```

만약 위의 방법을 사용한다면 이렇게 말이죠!

Reply:
오타방지가 가능하겠네요! 한 번 써볼까요?

Reply:
이슈 뺐습니다 ! #53